crates/xilem_masonry/examples/mason.rs

```diff
@@ -39,14 +39,17 @@ fn app_logic(data: &mut AppData) -> impl MasonryView<AppData> {
 
 fn toggleable(data: &mut AppData) -> impl MasonryView<AppData> {
     let inner_view: BoxedMasonryView<_, _> = if data.active {
-        Box::new(flex((
-            button("Deactivate", |data: &mut AppData| {
-                data.active = false;
-            }),
-            button("Unlimited Power", |data: &mut AppData| {
-                data.count = -1_000_000;
-            }),
-        )))
+        Box::new(
+            flex((
+                button("Deactivate", |data: &mut AppData| {
+                    data.active = false;
+                }),
+                button("Unlimited Power", |data: &mut AppData| {
+                    data.count = -1_000_000;
+                }),
+            ))
+            .direction(Axis::Horizontal),
```

Comment:
Should we have a dynamically toggling axis in the same widget here?

Reply:
Not sure, it jumps around the layout, which I didn't find satisfying, but I added it for the `+n` buttons when `data.active` changes, which is less annoying.